civil/civil.go

```diff
@@ -116,6 +127,24 @@ func (d Date) IsZero() bool {
 	return (d.Year == 0) && (int(d.Month) == 0) && (d.Day == 0)
 }
 
+// ISOWeek returns the ISO 8601 year and week number in which t occurs. Week
```

Comment:
Nit: feel free to do in your own commit
```suggestion
// ISOWeek returns the ISO 8601 year and week number in which d occurs. Week
```